src/pages/SignUp/SignUp.jsx

```diff
@@ -57,7 +146,7 @@ const SignUp = ({ login }) => {
         </div>
 
         <div className="signUp__continue">
-          <Button buttonText="Continue >" />
+          <Button buttonText="Continue >" handleClick={register} />
```

Comment:
Once the user has successfully created and account they should be taken to the sign in page.

Reply:
By default when this is completed the user will be signed in automatically. Should we take them to the Dashboard instead?